packages/php-wasm/compile/php/php_wasm.c

```diff
@@ -90,8 +91,10 @@ EMSCRIPTEN_KEEPALIVE FILE *wasm_popen(const char *cmd, const char *mode)
 		int current_procopen_call_id = ++procopen_call_id;
 		char *device_path = js_create_input_device(current_procopen_call_id);
 		int stdin_childend = current_procopen_call_id;
+        int stdin_parentend = open(device_path, O_WRONLY);
```

Comment:
Interesting! What does this line change?

Reply:
Actually, could the `open()` call be avoided by assigning NULL instead of the `stdin_parentend`?

Reply:
Yep ! I tried it right now and it works without that unused `stdin_parentend`.